pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go

```diff
@@ -366,6 +366,10 @@ func TestMostAllocatedScoringStrategy(t *testing.T) {
 
 			var gotScores framework.NodeScoreList
 			for _, n := range test.nodes {
+				status := p.(framework.PreScorePlugin).PreScore(ctx, state, test.requestedPod, test.nodes)
```

Comment:
Probably this should be moved to L366? (I mean we probably don't need to call PreScore in this `for` loop, right?)
can say the same thing for other tests.